molgenis/molgenis-emx2
# fix: load logo

Logo was not shown anymore, probably due to change in vite asssetloader behaviour  

need to test in PROD build context

Closes #3470 

how to test:
- go to the catalogue as check the specified ( or default) logo is show 
- to test with other logo use the debug option ( ?logo=UMCGkort.woordbeeld.svg )

todo:
